pyscf/adc/radc.py

```diff
@@ -654,17 +615,24 @@ def __init__(self, mf, frozen=0, mo_coeff=None, mo_occ=None):
         self.method = "adc(2)"
         self.method_type = "ip"
         self.with_df = None
-        self.compute_mpn_energy = True
-
-        keys = set(('conv_tol', 'e_corr', 'method', 'mo_coeff', 'mol', 'mo_energy', 'max_memory', 'incore_complete', 'scf_energy', 'e_tot', 't1', 'frozen', 'chkfile', 'max_space', 't2', 'mo_occ', 'max_cycle'))
+        self.compute_properties = True
+        self.evec_print_tol = 0.05
+        self.spec_thresh = 1e-8
```

Comment:
These default parameters are too small. I would set evec_print_tol to 0.1 and same for spec_thresh. Also, please rename spec_thresh to spec_factor_print_tol and apply all of these changes to both RADC and UADC.

Reply:
Done